src/test/e2e/services/environment-service.test.ts

```diff
@@ -136,3 +136,99 @@ test('Trying to get an environment that does not exist throws NotFoundError', as
         new NotFoundError(`Could not find environment with name: ${envName}`),
     );
 });
+
+test('Setting an override disables all other envs', async () => {
```

Comment:
I would love to see a similar test just geared towards the default environment. It should require "default" to be enabled and validate that built in envs (development and production) are disabled.

Reply:
Thanks, good suggestion, I'll add that!